tests/unit/amo/server/test_base.js

```diff
@@ -154,6 +155,155 @@ describe(__filename, () => {
   });
 
   describe('app', () => {
+    it('preconnects to addons-server CDN', async () => {
+      const config = getFakeConfig(
+        { staticHost: undefined, amoCDN: 'https://cdn.example.com' },
+        { allowUnknownKeys: true },
```

Comment:
Which of the two keys is unknown?

Reply:
we shouldn't have to use `allowUnknownKeys` here. `staticHost` declared in the main/default config.

Reply:
I just tried and I don't think you need any of that.

Reply:
Ah, my bad, I had an old config file laying around - and I wasn't seeing the warning about it because I was running `jest` directly.